properties.schema

```diff
@@ -31,23 +31,77 @@
                   "validators": [],
                   "help": "Turns Background Selector on and off. Acceptable values are true and false."
                 },
-                "_hideOnMobile": {
-                  "type":"boolean",
-                  "required":true,
-                  "default": false,
-                  "title": "Enabled",
-                  "inputType": { "type": "Boolean", "options": [false, true]},
-                  "validators": [],
-                  "help": "Hides the background image on small devices."
+                "_large": {
+                  "type":"object",
+                  "required": false,
+                  "title": "Desktop",
+                  "properties":{
+                    "_isEnabled": {
```

Comment:
Don't think we need to have a _isEnabled selector for each graphic. Should check in JS is _src is populated instead.